composer.json

```diff
@@ -150,6 +150,7 @@
         "release": "vendor/bin/monorepo-builder release patch --ansi"
     },
     "extra": {
+        "enable-patching": true,
         "patches": {
```

Comment:
Could you send this in standalone PR? It's no brainer to merge 👍

Reply:
Extracted to #1929